CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.143.3, 23 April 2021
+
+- Common: Transform update_types in IgnoreCondition
```

Comment:
This isn't really true: `do not transform update_types in IgnoreCondition` might be more correct.

Meh as it's internal frou-frou.

Reply:
Ah yes of course! 🙈